go.mod

```diff
@@ -16,6 +16,8 @@ require (
 	gotest.tools/gotestsum v1.9.0
 )
 
+replace github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.2 => github.com/vmanilo/terraform-plugin-sdk/v2 v2.25.2
```

Comment:
I dont want to use a fork of the SDK so we need to find away around this

Reply:
I've added PR to the terraform-plugin-sdk https://github.com/hashicorp/terraform-plugin-sdk/pull/1159

let's see if it be excepted and released